lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -798,8 +800,11 @@ sub _add_dependency_to_node {
     elsif ($dependency_type eq OpenQA::JobDependencies::Constants::PARALLEL) {
         $key = 'parallel_with';
     }
+    elsif ($dependency_type eq OpenQA::JobDependencies::Constants::DIRECTLY_CHAINED) {
+        $key = 'start_directly_after';
+    }
     else {
-        return;
+        return undef;
     }
```

Comment:
how about a mapping hash?:

```perl
my %dependency_to_key = (
    OpenQA::JobDependencies::Constants::CHAINED => 'start_after',
    OpenQA::JobDependencies::Constants::PARALLEL => 'parallel_with',
    OpenQA::JobDependencies::Constants::DIRECTLY_CHAINED => 'start_directly_after',
);
return $dependency_to_key{$key};
```

Reply:
Instead of adding a 3rd hash for this mapping I changed it to use the one defined in the constants.